docs/make.jl

```diff
@@ -248,8 +248,8 @@ MultiDocumenter.make(
         index_versions = ["stable"],
         engine = MultiDocumenter.FlexSearch
     ),
-    rootpath = "/",
-    canonical_domain = "https://JuliaAstro.org",
+    rootpath = "/JuliaAstro.org/",
```

Comment:
hmm, no dice just yet. Will see if changing to the PR preview root for this specific case moves the needle at all

Reply:
Yep, think that was the issue. For future Ian, checking network activity is your friend for debugging issues like these

![image](https://github.com/user-attachments/assets/357a7a31-3ea2-49de-997f-9b7e8b86cf66)